src/olympia/activity/admin.py

```diff
@@ -31,4 +32,52 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+class ReviewActionReasonLogAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
+    fields = (
+        'created',
+        'activity_log',
+        'user_email',
+        'reason',
+    )
+    list_display = (
+        'created',
+        'activity_log',
+        'reason',
+        'user_email',
+    )
+    list_filter = ('reason',)
+    readonly_fields = (
+        'created',
+        'activity_log',
+        'user_email',
+    )
+    search_fields = ('activity_log__user__email',)
+    view_on_site = False
+
+    @admin.display()
+    def user_email(self, obj):
+        return obj.activity_log.user.email
+
+    def has_add_permission(self, request):
```

Comment:
Might be worth checking what users will be expected to have access to this tool.  By default only superpowered admins can even view it (`*:*`) - if any other admin classes will need access to view/change then permissions need defining in olympia.constants/permissions.py

Reply:
This question has spurred some discussion about the validity of using only the admin for this feature, so for the purposes of this patch, I am going to keep it as is, and only allow super users to access the feature. We can have a follow-up if needed.